addons/medical/CfgVehicles.hpp

```diff
@@ -86,6 +86,18 @@ class CfgVehicles {
                 typeName = "NUMBER";
                 defaultValue = 1;
             };
+            class maxUnconsciousTime {
+                displayName = "Base Maximum Unconscious Time";
+                description = "The base amount of time that a unit will spend unconscious before waking up without medical intervention."; // TODO: Stringtable
+                typeName = "NUMBER";
+                defaultValue = 10; // testing value: 10
+            };
+            class maxDynamicUnconsciousTime {
```

Comment:
I don't see a good enough use for something like this to justify adding another setting for this. I feel it would only make things more confusing for users.